plat/rpi/rpi5/scmi/scmi.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#include <assert.h>
+#include <stdint.h>
+
+#include <drivers/scmi-msg.h>
+#include <drivers/scmi.h>
+
+#include <platform_def.h>
+
+#define RPI_SHM_BASE		RPI_SCMI_SHMEM_BASE
+#define RPI_SHM0_BASE	RPI_SHM_BASE
+
+static struct scmi_msg_channel scmi_channel[] = {
+	[0] = {
+		.shm_addr = RPI_SHM0_BASE,
+		.shm_size = SMT_BUF_SLOT_SIZE,
+	},
+};
+
+struct scmi_msg_channel *plat_scmi_get_channel(unsigned int agent_id)
+{
+	assert(agent_id < ARRAY_SIZE(scmi_channel));
+
+	return &scmi_channel[agent_id];
+}
+
+static const char vendor[] = "RPI5";
```

Comment:
Is this correct? I believe that vendor is either EPAM, Broadcom or RPi Foundation...

I believe, "EPAM" will be most correct

Reply:
yeah... right )))